### ISSUE TITLE
ods:taxonFormula: unclear definition and location of the term

### ISSUE THREAD

**Issue by DavidFichtmueller** [DiSSCo/openDS#167]
### Term Name

ods:taxonFormula

### Digital Object Name

DigitalSpecimen: Identification

### Feedback

What is meant with "full formular" in the context of the taxonomic identification. 

Also if this term pertains to the taxonomic identification, shouldn't it be located within the class ods:TaxonomicIdentification and not at this class (ods:Identification)?

**Comment by samleeflang** [comment]
Agree, for now we are removing this term. Probably we could use `ods:specimenName` for the taxonformula as it is basically the combined taxonIdentificaitons (in cases where one specimen consists of multiple taxa). If the need requires us to add this are a similar term we can always add it. For more on taxonFormula also see what I answered in issue https://github.com/DiSSCo/openDS/issues/170
